src2/app.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express');
+const cookieParser = require('cookie-parser');
+const router = express.Router();
+
+export const app = express();
+
+const products = [
+    {
+        name: 'product',
+        reviews: {
+            Nick: 'jkjkjlj',
+            Jan: 'jkjljlkjjkjjkjkj'
+        }
+    },
+    {
+        name: 'product1',
+        reviews: {
+            Nicky: 'jkjkjlj',
+            Jany: 'jkjljlkjjkjjkjkj'
+        }
+    }
+];
+
+const users = [
+    {
+        name: 'Vasya'
+    },
+    {
+        name: "Petya"
+    }
+];
+
+router.get('/api/products', function(req, res) {
```

Comment:
I'd rather create a separated file for routes and add them like this `app.use('/api', apiRoutes);`

Reply:
Using router rather than use was recommended in lecture. Also I've seen this architecture in tutorials. I can agree about separating this file in two, but I thought that in such a small project this wouldn't be critical.

Reply:
I've meant something like:
```javascript
import apiRoutes from './routes/api';

app.use('/api', apiRoutes);
```
I didn't offer you not to use a router :) What about architecture, anyway you can't predict the future size of your project that means you should build clear and maintainable architecture from the very beginning!

Reply:
Done